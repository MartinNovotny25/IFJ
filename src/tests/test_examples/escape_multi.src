a = '\A\z\z\'\\\'\z\""\'\z\'\\\\\z\x7A\m\m'
a= '\Z'
print(a)
print('\x61\X')

import random
alphanum_list = ["X","\\x", "\\\\",]
for letter in range(65,70):
    alphanum_list.append(chr(letter))

for number in range(0,9+1):
    alphanum_list.append(number)

for j in range(0,20):
    print('\'',end='')
    for i in range(0,20):
        a = "{}".format(random.choice(alphanum_list))
        print(a, end='')
    print('\'')
