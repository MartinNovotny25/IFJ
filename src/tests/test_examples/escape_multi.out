TOKEN(IDENTIFIKATOR)
TOKEN(=)
TOKEN(STRING, \A\z\z'\'\z""'\z'\\\zz\m\m)
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(=)
TOKEN(STRING, \Z)
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(IDENTIFIKATOR)
TOKEN())
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(STRING, a\X)
TOKEN())
TOKEN(EOL)
TOKEN(EOF)
